code/game/objects/items/battering_ram.dm

```diff
@@ -0,0 +1,117 @@
+/*
+ * BATTERING RAM
+ *
+ * KNOCK-KNOCK, its Justice!
+ */
+/obj/item/batteringram
+	icon = 'icons/obj/weapons/fireaxe.dmi'
+	icon_state = "battering_ram0"
+	base_icon_state = "battering_ram"
+	lefthand_file = 'icons/mob/inhands/weapons/axes_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/axes_righthand.dmi'
+	name = "battering ram"
+	desc = "An extremely heavy and unwieldy hunk of metal used to break down airlocks. Why would anyone ram a sliding airlock?"
+
+	w_class = WEIGHT_CLASS_HUGE
```

Comment:
Given carlarc suggested it should be a traitor objective, maybe make it bulky?

Reply:
ok but its not one?